### ISSUE TITLE
Example Request: PyTorch Precision Recall

### ISSUE THREAD

**Issue by FuriouslyCurious** [reiinakano/scikit-plot#45]
Thank you for the Keras example in README: scikit-plot looks like a very elegant solution for plotting ML / DL curves.  

I was wondering if you have tried plotting PyTorch optimizers with scikit-plot?  If you have an example of PyTorch it will help me out a lot.  

Thank you again for an awesome library!

**Comment by reiinakano** [comment]
What kind of graphs would you like to see from a Pytorch optimizer?

**Comment by FuriouslyCurious** [comment]
PyTorch has torch.optim.lr_scheduler.ReduceLROnPlateau() method which reduces learning rate (LR) when stuck on a plateau.  This method is similar to Keras method ReduceLROnPlateau().

A plot of LR for each Epoch would be a great example.  

Doc: http://pytorch.org/docs/master/optim.html#how-to-adjust-learning-rate

**Comment by reiinakano** [comment]
Hi @FuriouslyCurious, I am not sure if this is a problem that needs to be solved by scikit-plot. If you want to plot LR per epoch, then you need to collect LR and Epochs into two lists and just use vanilla `matplotlib` to plot the curve. `scikit-plot` is mostly for curves that require additional computation, and I don't see how a plot of LR per epoch fits within scikit-plot.
